tests/cli/BuildCest.php

```diff
@@ -199,4 +199,224 @@ public function noReturnForNeverType(CliGuy $I, Scenario $scenario)
         $I->seeInThisFile('$this->getScenario()->runStep(new \Codeception\Step\ConditionalAssertion(\'seeDirFound\', func_get_args()));');
         $I->dontSeeInThisFile('return $this->getScenario()->runStep(new \Codeception\Step\ConditionalAssertion(\'seeDirFound\', func_get_args()));');
     }
+
+    public function generateAttributeForMethodAttributeWithNoParametersAndNoBrackets(CliGuy $I, Scenario $scenario): void
+    {
+        if (PHP_MAJOR_VERSION < 8) {
+            $scenario->skip('Does not work in PHP < 8');
```

Comment:
Skips for `< ū` and `<7` aren't necessaty, because the lowest supported version is 8.0.
It is only necessary to skip tests that require PHP 8.1 or 8.2,